pkg/model/node.go

```diff
@@ -285,6 +286,14 @@ func (n *Node) HasPrice() bool {
 }
 
 func (n *Node) UpdatePrice(pricing *pricing.Provider) {
+	// If the node has the instance-price override label, don't look up pricing
```

Comment:
Seems fine to me, wonder if the label name should be different? I've got no better ideas though. LGTM unless you come up with a better name.

Reply:
Yeah I was wondering if we should make the label group karpenter.sh, but figured it might have more use-cases outside of just Karpenter KwoK. Maybe something like `node-viewer/instance-price-override`?

Reply:
That works for me.